app/RockTheVoteRecord.php

```diff
@@ -58,15 +58,18 @@ public function __construct($record, $config = null)
                 'email_subscription_topics' => $emailOptIn ? explode(',', $config['user']['email_subscription_topics']) : [],
                 'first_name' => $record['First name'],
                 'last_name' => $record['Last name'],
-                'mobile' => isset($record[static::$mobileFieldName]) && is_valid_mobile($record[static::$mobileFieldName]) ? $record[static::$mobileFieldName] : null,
+                'mobile' => null,
+                'sms_status' => null,
             ]);
 
-            // If a mobile was provided, set SMS subscription per opt-in value.
-            if ($this->userData['mobile']) {
-                $smsOptIn = str_to_boolean($record[static::$smsOptInFieldName]);
-
-                $this->userData['sms_status'] = $smsOptIn ? SmsStatus::$active : SmsStatus::$stop;
-                $this->userData['sms_subscription_topics'] = $smsOptIn ? explode(',', $config['user']['sms_subscription_topics']) : [];
+            // If a the user opted in to SMS messaging, add their mobile number and SMS subscription fields
+            $smsOptIn = str_to_boolean($record[static::$smsOptInFieldName]);
+            if ($smsOptIn) {
+                if (isset($record[static::$mobileFieldName]) && is_valid_mobile($record[static::$mobileFieldName])) {
```

Comment:
This is nitpicky and optional, but I've been trying to keep line breaks and conditionals per our [communal docs](https://github.com/DoSomething/communal-docs/tree/master/Contributing#line-breaks) when adding/updating code in this repo. I think a single conditional could help simplify?

```suggestion
            $smsOptIn = str_to_boolean($record[static::$smsOptInFieldName]);

            if ($smsOptIn && isset($record[static::$mobileFieldName]) && is_valid_mobile($record[static::$mobileFieldName])) {
```

Reply:
ah good call on the spacing! I like combining them too, previously I had this logic less streamlined but now could totally do all in one line like that!